roles/StackStorm.st2/tasks/proxy.yml

```diff
@@ -10,7 +10,7 @@
     # NB: Empty ENV var cast to 'None' string in Ansible
     state: "{{ 'present' if ansible_facts.env.get(item.1, 'None') != 'None' else 'absent' }}"
   vars:
-    _services: [st2api, st2actionrunner]
+    _services: [st2api, st2actionrunner, st2chatops]
```

Comment:
After looking closer, this addition should belong to `st2chatops` Ansible role as it could be installed separately.

Reply:
Ok, how can we do this when this one has already been merged?

Reply:
Just open another PR that adds to `st2chatops` and removes from `st2`